public/javascripts/gformFileUpload.js

```diff
@@ -40,7 +40,7 @@
           url: frm.attr('action'),
           data: frm.serialize()
       }).then(function (){
-          frm.submit();
+          submitButton.submit();
```

Comment:
Testing this locally it seems not to work. Are you confident this is correct solution?